client/src/endpoint/mod.rs

```diff
@@ -4,12 +4,10 @@ use error::Result;
 use serde::de::DeserializeOwned;
 use http;
 
-mod account;
-mod asset;
+pub mod account;
+pub mod asset;
```

Comment:
Why don't we need any documentation here?  Looks like things are silenced here https://github.com/kbacha/stellar-sdk/blob/master/client/src/lib.rs#L1  any particular reason for this?

Reply:
It's not silenced. I moved the documentation to the header of the asset and account mods
https://github.com/kbacha/stellar-sdk/pull/69/files#diff-a7ef25ce71dce3bbe0f8bed2034b991cR1